apps/andi/lib/andi_web/live/edit_live_view/data_dictionary_form.ex

```diff
@@ -194,6 +303,54 @@ defmodule AndiWeb.EditLiveView.DataDictionaryForm do
     {:noreply, assign(socket, current_data_dictionary_item: field, selected_field_id: field_id)}
   end
 
+  defp parse_csv(file_string) do
```

Comment:
Did you consider using a library for this? Does it perform decently well for large dataset files?

Reply:
This function does perform well, but that brings up a good point about other steps performing well